### ISSUE TITLE
remove the parallax

### ISSUE THREAD

**Issue by williamsherlock** [iamhexcoder/parallax-element#1]
how can i remove the parallax element?

**Comment by baerkins** [comment]
Hi @williamsherlock 

There currently is not a destroy method, but I will add it as a milestone for the next release. You can however remove the parallax on mobile devices, just set the option `disableMobile` to `true`:

```
$('.element').parallaxElement({
  disableMobile: true
});
```

Not sure if that helps or not.

Are you looking to remove parallax from a single element or from all elements?

**Comment by williamsherlock** [comment]
dear @iamhexcoder 

i wanna to re-initialize the parallax speed, if i add 0.2 and the 0.4 its run twice instead of once.

is there any solution to solve this?

**Comment by baerkins** [comment]
@williamsherlock 

You could write a function that changes the `data-speed` value on an event. Something like

```
$('.button').on('click', function() {
 $('#el-to-change').attr('data-speed', '0.4');
});
```

I haven't tested this at all, but `data-speed` is just a number by which a translate value is calculated on scroll, so in theory it should work.

**Comment by williamsherlock** [comment]
@iamhexcoder 
yes once it is initial we need to only change $('#el').attr('data-speed', '0.4'); instead of reinitialize. :+1: 

thanks!
